app/assets/javascripts/messages.js

```diff
@@ -1,36 +1,17 @@
 $(document).ready(function () {
-  $(".inbox-mark-unread").on("ajax:success", function (event, data) {
-    updateHtml(data);
-    updateReadState(this, false);
+  $(".messages-table .inbox-destroy").on("turbo:submit-end", function (event) {
```

Comment:
There is no need for jQuery here. I just kept it for consistency but would love to remove it here.

Reply:
See I have a visceral hatred of doing stuff with raw DOM methods and much prefer jQuery so I suspect that is a separate discussion the project is going to need to have...